### ISSUE TITLE
Fixing W503 line break before binary operator puts `and` within comment

### ISSUE THREAD

**Issue by taldcroft** [hhatto/autopep8#503]
---

## Python Code
```python

if (True  #
    and True
    and True):
    print(1)
```

## Command Line and Configuration

Command Line
```shell
$ autopep8 -d --select=W503 junk.py 
--- original/junk.py
+++ fixed/junk.py
@@ -1,4 +1,4 @@
-if (True  #
-    and True
-    and True):
+if (True  # and
+    True and
+    True):
     print(1)
```

## Your Environment
* Python version: Python 3.6.8 :: Anaconda, Inc.
* autopep8 version: autopep8 1.5a1 (pycodestyle: 2.5.0)
* Platform: macOSX

**Comment by hhatto** [comment]
this is fix in #511 .

I plan release to version 1.5.

**Comment by taldcroft** [comment]
Thanks!  🥇
